gdk4-macos/src/macos_surface.rs

```diff
@@ -0,0 +1,26 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+#[cfg(feature = "v4_8")]
+use crate::ffi;
+use crate::MacosSurface;
+#[cfg(feature = "v4_8")]
+use cocoa::base::id;
+#[cfg(feature = "v4_8")]
+use glib::translate::*;
+
+impl MacosSurface {
+    #[cfg(feature = "v4_8")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "v4_8")))]
+    #[doc(alias = "gdk_macos_surface_get_native_window")]
+    #[doc(alias = "get_native_window")]
+    pub fn native(&self) -> Option<id> {
```

Comment:
This still doesn't have two fallbacks.
- If the cocoa feature is disabled, you should have this function that gives you access to the raw pointer.
- If you don't have v4_8 enabled, this function should go through ObjectExt::property and give you the native window.

Reply:
I think this should be now ready